client/assets/pages/_events-page.scss

```diff
@@ -338,6 +338,8 @@ main {
 .cd-header-title {
   vertical-align: middle;
   display: table-cell;
+  font-size: 14px;
+  color: #a2a2a2
```

Comment:
Also I think this hex code already exists in the `variables.scss` file so you can just access the variable name instead. 

@kenneth-kip I think we use upper-case hex code (`#A2A2A2` instead of `#a2a2a2`), kindly confirm